tests/integration/component.go

```diff
@@ -712,4 +712,67 @@ func componentTests(args ...string) {
 			Expect(stdError).To(ContainSubstring("quantities must match the regular expression"))
 		})
 	})
+
+	Context("Creating component using symlink", func() {
+		var symLinkPath string
+
+		JustBeforeEach(func() {
+			if runtime.GOOS == "windows" {
```

Comment:
PS C:\Users\Admin\go\src\github.com\openshift\odo> make test-cmd-cmp
The system cannot find the path specified.
'grep' is not recognized as an internal or external command,
operable program or batch file.
ginkgo -v -randomizeAllSpecs -slowSpecThreshold=120 -timeout 7200s -nodes=1 -focus="odo component command tests" tests/integration/
Running Suite: Integration Suite
================================
Random Seed: 1572342873 - Will randomize all specs
Will run 2 of 55 specs

SSSSSSSSSSSSSSSSSS
------------------------------
odo component command tests Creating component using symlink 
  Should be able to deploy a wildfly war file using symlinks in some odo commands
  C:/Users/Admin/go/src/github.com/openshift/odo/tests/integration/component.go:763
Created dir: C:\Users\Admin\AppData\Local\Temp\921376195
Deleting dir: C:\Users\Admin\AppData\Local\Temp\921376195

S [SKIPPING] in Spec Setup (JustBeforeEach) [0.004 seconds]
odo component command tests
C:/Users/Admin/go/src/github.com/openshift/odo/tests/integration/cmd_cmp_test.go:13
  Creating component using symlink
  C:/Users/Admin/go/src/github.com/openshift/odo/tests/integration/component.go:718
    Should be able to deploy a wildfly war file using symlinks in some odo commands [JustBeforeEach]
    C:/Users/Admin/go/src/github.com/openshift/odo/tests/integration/component.go:763

    skipping on windows because of admin issue in go library

    C:/Users/Admin/go/src/github.com/openshift/odo/tests/integration/component.go:723
------------------------------
SSSSSSSSSSSS
------------------------------
odo component command tests Creating component using symlink 
    C:/Users/Admin/go/src/github.com/openshift/odo/tests/integration/component.go:742

    skipping on windows because of admin issue in go library

    C:/Users/Admin/go/src/github.com/openshift/odo/tests/integration/component.go:723
------------------------------
SSSSSSSSSSSSSSSSSSSSSSS
Ran 0 of 55 Specs in 0.914 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 55 Skipped
PASS

Ginkgo ran 1 suite in 6.150263s
Test Suite Passed
PS C:\Users\Admin\go\src\github.com\openshift\odo>
/approve